src/test/groovy/com/tidyjava/bp/PostControllerSpec.groovy

```diff
@@ -0,0 +1,66 @@
+package com.tidyjava.bp
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.SpringApplicationConfiguration
+import org.springframework.test.context.web.WebAppConfiguration
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.setup.MockMvcBuilders
+import org.springframework.web.context.WebApplicationContext
+import spock.lang.Specification
+
+import static java.time.format.DateTimeFormatter.ISO_LOCAL_DATE
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.view
+
+@SpringApplicationConfiguration(BloggingPlatform.class)
+@WebAppConfiguration
+class PostControllerSpec extends Specification {
```

Comment:
If this is integration test, maybe it will be good to name class like PostControllerIntegrationSpec?